test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -959,7 +959,8 @@
                       first))))))))
 
 (deftest datetime-diff-type-test
-  (mt/test-drivers (->> (disj (mt/normal-drivers-with-feature :datetime-diff) :snowflake)
+  ;; FIXME: time fields in sqlite aren't wrapped with a TypedHoneySQLForm, this test fails
```

Comment:
Did you try to investigate why this is the case? It probably would have been easy to just fix that stuff there

It seems like the impls of `->honeysql` for `LocalTime` and `OffsetTime` are missing the type info:

```clj
(defmethod sql.qp/->honeysql [:sqlite LocalTime]
  [_ t]
  (hsql/call :time (hx/literal (u.date/format-sql t))))
``` 

should be an easy fix, right?